src/Tools/dotnet-monitor/Auth/ApiAuthenticationOptions.cs

```diff
@@ -10,7 +13,14 @@ namespace Microsoft.Diagnostics.Tools.Monitor
 {
     internal sealed class ApiAuthenticationOptions
     {
+        [Description("API Key in Hashed form. Each byte should be two hex based digits.")]
```

Comment:
nit: "key" and "hashed" should not be capitalized. And "hex-based" (note hyphen). Maybe "hexadecimal" instead of "hex". For example:

"The API key in hashed form. Each byte should be two hexadecimal-based digits."